docs/import.md

```diff
@@ -1,49 +1,71 @@
-# Import
 
-GGUF models and select Safetensors models can be imported directly into Ollama.
+# Importing a model
 
-## Import GGUF
+You can import a model into Ollama;
 
-A binary GGUF file can be imported directly into Ollama through a Modelfile.
+  * From Safetensors weights; or
 
-```dockerfile
-FROM /path/to/file.gguf
-```
+  * From a GGUF file
+
+## Importing from Safetensors weights
+
+Importing your model directly from the Safetensors weights is the easiest way to
+import a model. Ollama supports importing several different architectures including:
 
-## Import Safetensors
+  * Llama (including Llama 2, Llama 3, and Llama 3.1);
+  * Mistral (including Mistral 1, Mistral 2, and Mixtral); and
+  * Gemma (including Gemma 1 and Gemma 2)
 
-If the model being imported is one of these architectures, it can be imported directly into Ollama through a Modelfile:
+This includes importing either the foundation model as well as any fine tuned model which
+is based upon one of these architectures.
 
- - LlamaForCausalLM
- - MistralForCausalLM
- - GemmaForCausalLM
+To import a model, create a Modelfile using a text editor which includes a `FROM`
+line which points to the directory with the model weights.
 
 ```dockerfile
```

Comment:
do we want to break this out into:

1. Step 1: Write a Modelfile
2. Step 2: Run `ollama create`

Reply:
I really like this suggestion.